functions.php

```diff
@@ -3223,5 +3223,78 @@ function add_classic_degree_body_class( $classes ) {
 
 add_action( 'body_class', 'add_classic_degree_body_class' );
 
+function add_colleges_fields( $taxonomy ) {
```

Comment:
Consider moving the college meta-related functions up in this file, next to the other college functions (lines 2521-2667) for organization/clarity.
https://github.com/UCF/Main-Site-Theme/blob/33494a87139882a23952a474007fb8c04dde9940/functions.php#L2522